File: TLOUdle.Server/Modulos/Personajes.cs
using Microsoft.Identity.Client;

namespace TLOUdle.Server.Modulos
{
    public class Personajes
    {
        public int Id { get; set; }
        public String Nombre { get; set; }
        public String Altura { get; set; }
        public int Edad {  get; set; }
        public String Sexo { get; set; }
        public String ColorPelo { get; set; }
        public int Juegos { get; set; }
        public String TipoPersonaje { get; set; }
        public String Estado { get; set; }
        public String Descripcion { get; set; }
        public String Foto { get; set; }
    }
}

File: tloudle.client/src/app/Interfaces/personajes.ts
//Interfaz Para la tabal de perosnajes
export interface Personajes {
  id?: number,
  nombre?: string,
  altura: number,
  edad?: number,
  sexo: string,
  colorPelo: string,
  juegos?: number,
  tipoPersonaje: string,
  estado: string,
  descripcion: string,
  foto: string
}
